com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfRobotExtensions.cs

```diff
@@ -322,14 +335,16 @@ private static Robot ExportRobotData(this UrdfRobot urdfRobot)
                         "Ok");
                     return null;
                 }
+
                 robot.links.Add(urdfLink.ExportLinkData());
                 linkNames.Add(urdfLink.name);
 
-                //Joint export
+                //Joints export
                 UrdfJoint urdfJoint = urdfLink.gameObject.GetComponent<UrdfJoint>();
                 if (urdfJoint != null)
                     robot.joints.Add(urdfJoint.ExportJointData());
                 else if (!urdfLink.IsBaseLink)
+
```

Comment:
remove black line. might also be good to enclose the else body in brackets as it is more than 1 line even though one is a comment.